src/async_graphql_hyper.rs

```diff
@@ -17,6 +17,13 @@ impl GraphQLBatchRequest {
   {
     GraphQLResponse(executor.execute_batch(self.0).await)
   }
+
+  pub fn data<D: Any + Clone + Send + Sync>(mut self, data: D) -> Self {
```

Comment:
Are we using this function anywhere?

Reply:
Yes, in `graphql_batch_request` in  `server.rs`